exporter/clickhouseexporter/exporter_traces.go

```diff
@@ -196,9 +197,10 @@ CREATE TABLE IF NOT EXISTS %s %s (
      INDEX idx_span_attr_value mapValues(SpanAttributes) TYPE bloom_filter(0.01) GRANULARITY 1,
      INDEX idx_duration Duration TYPE minmax GRANULARITY 1
 ) ENGINE = %s
+PARTITION BY toDate(TimestampTime)
+PRIMARY KEY (ServiceName, SpanName, TimestampTime)
+ORDER BY (ServiceName, SpanName, TimestampTime, Timestamp, TraceId)
```

Comment:
Is this change considered a breaking change?

Reply:
Nope, exporter only cares about `INSERT`, and no columns have changed. This is an optimization for new tables